src/common/environment-helper.ts

```diff
@@ -0,0 +1,17 @@
+// production environment= "production"
+// local development = "development"
+// preview release = "staging"
+// feature branch (github PRs)= "feature"
+// test environment = "test"
+
+import { IS_TEST_ENV } from './constants';
+
+function getEnvironment() {
```

Comment:
@kyranjamie @fbwoolf I added this helper so we can easily check what kind of environment we are in.

Reply:
Nice one @beguene, this is helpful function